CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ Starting from version 0.20.1, this format is based on [Keep a Changelog], and th
 to [Semantic Versioning]. Full commit history is available in the
 [commit logs](https://github.com/scverse/scvi-tools/commits/).
 
-## Version 1.2
+## Version 1.3
 
```

Comment:
Remove this from the PR. It shouldn't be part of 1.2.X, whereas the rest of the commit should.